src/EPR.Payment.Facade.Common/Dtos/Request/Payments/CompletePaymentRequestDto.cs

```diff
@@ -1,17 +1,14 @@
-﻿using Newtonsoft.Json;
```

Comment:
can we delete this file if the class is not used

Reply:
Sure, sorry, meant to do this. I will delete.